packages/banner/src/components/LidoStakeBanner.tsx

```diff
@@ -1,9 +1,10 @@
 import React, { useEffect, useState } from 'react';
 import { Button, Box, Text, Link, useBreakpoint } from '@lidofinance/lido-ui';
 
-import { getStakeApy, STATIC_DEFAULT_APY } from '@/api/stakeApy';
+import { getStakeApy, STATIC_DEFAULT_APY } from '@common/stakeApy';
```

Comment:
maybe it's better to do `common` folder as a separated package?
and for example put there some other heplers (just a suggestion for discussion)

Reply:
There is no too much files for making it separated package, It's not worthy